src/test/java/io/strimzi/kafka/quotas/VolumeSourceTest.java

```diff
@@ -228,6 +350,75 @@ void shouldProduceEmptyIfDescribeLogDirsReturnsEmptyMaps() {
         assertThat(onlyInvocation.getVolumeUsages()).isEmpty();
     }
 
+    @Test
+    void shouldCountEachActiveBrokerInDescribeClusterResponse() {
+        //Given
+        final int nodeId = 1;
+        final int node2Id = nodeId + 1;
+        givenNode(nodeId);
+        givenNode(node2Id);
+        givenLogDirDescription(nodeId, "dir1", 50, 10);
+        givenLogDirDescription(nodeId, "dir2", 60, 15);
+        givenLogDirDescription(node2Id, "dir3", 40, 1);
+
+        //When
+        volumeSource.run();
+
+        //Then
+        final SortedMap<MetricName, Metric> volumeSourceMetrics = getMetricGroup(METRICS_SCOPE, METRICS_TYPE);
+        assertGaugeMetric(volumeSourceMetrics, "ActiveBrokers", buildBasicTagMap(), 2L);
+    }
+
+    @Test
+    void shouldCountEachActiveLogDirInDescribeLogDirsResponse() {
+        //Given
+        final int nodeId = 1;
+        final int node2Id = nodeId + 1;
+        givenNode(nodeId);
+        givenNode(node2Id);
+        givenLogDirDescription(nodeId, "dir1", 50, 10);
+        givenLogDirDescription(nodeId, "dir2", 60, 15);
+        givenLogDirDescription(node2Id, "dir3", 40, 1);
+
+        //When
+        volumeSource.run();
+
+        //Then
+        final SortedMap<MetricName, Metric> volumeSourceMetrics = getMetricGroup(METRICS_SCOPE, METRICS_TYPE);
+        assertGaugeMetric(volumeSourceMetrics, "ActiveLogDirs", buildBasicTagMap(), 3L);
+    }
+
+    @Test
+    void shouldCountEachValidLogDirInDescribeLogDirsResponse() {
+        //Given
+        final int nodeId = 1;
+        final int node2Id = nodeId + 1;
```

Comment:
nit: I can see in most cases, the nodeId is 1, node2Id = 2, maybe we can put them as class variable.

Reply:
I had left them in the tests with the idea of being explicit in context but now you mention it they probably should be fields.

Reply:
Addressed in 738b43120f8e0b5621a84d5c844e000d873a5a5e